### PR TITLE
extracted changes from sdk-redux re sdk-core

### PR DESCRIPTION
**Pull request by kasparkallas**

### CODE REVIEW

File: packages/sdk-core/test/setup.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import { ethers } from "hardhat";
```
Comment (0xdavinchee) [inline packages/sdk-core/test/setup.ts:1]: please delete this file

**Review by 0xdavinchee — COMMENTED** [review]
looks good, pls just delete the `test/setup.ts` file.
nice solution w/ the typedoc too btw!

File: packages/sdk-core/test/setup.ts (hunk 1 above)
Comment (hellwolf) [reply packages/sdk-core/test/setup.ts]: deleted

**Review by 0xdavinchee — APPROVED** [review]
gmi

**Pull request merged**